frontend/src/metabase/setup/selectors.ts

```diff
@@ -21,6 +21,10 @@ export const getUserEmail = (state: State): string | undefined => {
   return getUser(state)?.email;
 };
 
+export const getUsageReason = (state: State) => {
```

Comment:
I assume the usage reason will land somewhere else once we decide we're not going to store it in Redux? And by then, this selector along with its counterparts will be removed as well, is that right?

Reply:
Ah I was confused. Ignore it then.